src/CodeGen_PyTorch.cpp

```diff
@@ -310,6 +310,14 @@ inline int test1_th_(at::Tensor &_buf, float _alpha, int32_t _beta) {
         compare_src(src.str(), correct_src);
     }
 
+    // TODO: host_supports_target_device('cuda') crashes on arm32 inside MCJIT
+    // (at least on our buildbots). Skip there pending investigation.
+    // https://github.com/halide/Halide/issues/4940
+    if (get_host_target().arch == Target::ARM && get_target().bits == 32) {
```

Comment:
s/get_target/get_host_target?

Reply:
Yeah, it's wrong... but I'm going to withdraw this PR; there are other tests broken similarly on arm32 and I need to either fix the underlying issue, or find a similar stopgap.